llama-index-integrations/tools/llama-index-tools-dappier/tests/test_tools_dappier_real_time_search.py

```
@@ -0,0 +1,68 @@
+
+def test_class():
+    names_of_base_classes = [b.__name__ for b in DappierRealTimeSearchToolSpec.__mro__]
+    assert BaseToolSpec.__name__ in names_of_base_classes
+
+
+@pytest.fixture()
+def dappier_client():
+    return MagicMock()
+
+
+@pytest.fixture()
```

Ruff (0.8.2)

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)